Assets/MixedRealityToolkit/InputSystem/Cursors/BaseCursor.cs

```diff
@@ -133,11 +133,9 @@ public bool IsVisible
         /// <param name="eventData"></param>
         public virtual void OnSourceDetected(SourceStateEventData eventData)
         {
+            var controller = eventData.Controller as WindowsMixedRealityController;
```

Comment:
shouldnt this line be inside the UNITY_WSA condition?

Reply:
No, because we've already wrapped it in the controller class.

Reply:
I can live with that, it was just a touch confusing at first